shared/src/commonMain/kotlin/ui/composables/DCQLCredentialQuerySubmissionSelectionOption.kt

```diff
@@ -91,6 +91,7 @@ fun DCQLCredentialQuerySubmissionSelectionOption(
         }
 
         override val representation = credential.representation
+        override val scheme = credential.scheme!!
```

Comment:
can this break when updating?

Reply:
It can't break, since we're using only local classes and objects that are being loaded freshly every time we start up the app.